content/blog/2017-01-tokio-0-1.md

```diff
@@ -20,15 +20,13 @@ servers like proxies.
 particular for high performance protocol implementations. Over time, we expect
 Tokio to grow a rich middleware ecosystem and ultimately to support various web
 and application frameworks. [Hyper], for example, has been adding Tokio
-integration, and there's a [growing list] of other protocol implementations as
+integration, and there's a growing list of other protocol implementations as
 well.
 
 [hyper]: http://hyper.rs/
 
-[growing list]: {{ relref "../community/third-party.md" }}
```

Comment:
i'm assuming this link was broken?

Reply:
Yeah. We also don't use this syntax for links anymore.